src/Config/Type.hs

```diff
@@ -119,11 +121,28 @@ instance Semigroup RestrictIdents where
     OnlyIdents x1 <> OnlyIdents x2 = OnlyIdents $ x1 <> x2
     ri1 <> ri2 = error $ "Incompatible restrictions: " ++ show (ri1, ri2)
 
+data RestrictImportStyle
+  = ImportStyleQualified
+  | ImportStyleUnqualified
+  | ImportStyleExplicit
+  | ImportStyleExplicitOrQualified
+  | ImportStyleIgnore
```

Comment:
I guess Ignore means "I don't care" - would `ImportStyleUnrestricted` be a better name?

Reply:
That said, if we want ignore as the API for users in YAML, this seems fine. I think I might be ambivalent